similar/main/switch.cpp

```diff
@@ -293,21 +299,26 @@ int check_trigger_sub(object &plrobj, const trgnum_t trigger_num, int pnum,int s
 			plrobj.shields -= trigger.value;
 		}
 
-		if (trigger.flags & TRIGGER_EXIT) {
-			start_endlevel_sequence();
+		if (trigger.flags & TRIGGER_EXIT)
+		{
+			result = start_endlevel_sequence();
+			if (result == window_event_result::handled)
+				result = window_event_result::ignored;	// call multi_send_trigger, or end game anyway
 		}
 
 		if (trigger.flags & TRIGGER_SECRET_EXIT) {
+			if (trigger.flags & TRIGGER_EXIT)
+				LevelError("Trigger %u is both a regular and secret exit! Not recommended", trigger_num);
```

Comment:
This will look a bit odd after string concatenation.  `LevelError` always appends to the text `"  Please report this to the level author, not to the Rebirth maintainers."`, so the bare `Not recommended` will be formatted oddly.  It might look better to say `  This is not a supported combination.` or `This combination is deprecated.`